### DISCUSSION TITLE
can someone activate realtime for tables using the cli

### DISCUSSION THREAD

**Discussion by joaomelo** [supabase/supabase#9674] [Questions] [answered]
Hi there,

My understanding is that realtime is not active by default. Is there a way to activate it programmatically using migrations or superbase CLI on the local environment?

I want to automate my project's tests in the CI/CD pipeline, and I can't use the studio UI for that scenario.

Thanks,
JM

**Comment by GaryAustin1** [comment, marked as answer]
This guide gives the sql needed to enable realtime on a table. 
https://supabase.com/docs/guides/realtime/postgres-cdc

**Reply by joaomelo** [reply]
Yeah. Tks very much!
